tests/virt_autotest/reboot_and_wait_up_normal1.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
```

Comment:
We may need multiple reboot step call in one test, but if write it in only one file, in your test list showed in the frontend test log, it will only show one. I have talked about this with kulow, but kulow said openqa will not change code to fix this problem. SO the solution is to use files with the same content to make them all shown in frontend log. So we add normal[num]. In future if we need the second normal reboot(just reboot, no upgrade or others), we will add another file reboot_and_wait_up_normal2.pm